packages/content/test/attributes/EmailAddress.test.ts

```diff
@@ -0,0 +1,59 @@
+import { ParsingError } from "@js-soft/ts-serval";
+import { EMailAddress } from "../../src";
+
+describe("creation of RepositoryAttributes of Attribute Value Type EMailAddress", function () {
+    test("can create a RepositoryAttribute of Attribute Value Type EMailAddress", function () {
+        const validEMailAdress = EMailAddress.from({
+            value: "peter123@inwind.it"
+        });
+        expect(validEMailAdress.value.toString()).toBe("peter123@inwind.it");
+    });
+    test("can create a RepositoryAttribute of Attribute Value Type EMailAddress with the German 'ä' in its domain", function () {
+        const validEMailAdress = EMailAddress.from({
+            value: "peter123@inwänd.it"
+        });
+        expect(validEMailAdress.value.toString()).toBe("peter123@inwänd.it");
+    });
+    test("returns an error when trying to create an Attribute Value Type EMailAddress with a blank in the value for EMailAddress", function () {
+        const invalidEMailAdressCall = () => {
+            EMailAddress.from({
+                value: "Hugo Becker@gmx.de"
+            });
+        };
+        expect(invalidEMailAdressCall).toThrow(
+            new ParsingError(
+                "EMailAddress",
+                "value",
+                "Value does not match regular expression /^[a-z0-9!#$%&'*+/=?^_`{|}~-]+(?:[.][a-z0-9!#$%&'*+/=?^_`{|}~-]+){0,}@(?:[a-z0-9ÄäÖöÜüß](?:[a-zÄäÖöÜüß0-9-]{0,61}[a-zÄäÖöÜüß0-9]){0,1}[.])+[a-zÄäÖöÜüß0-9](?:[a-zÄäÖöÜüß0-9-]{0,61}[a-zÄäÖöÜüß0-9])$/i"
+            )
+        );
+    });
+    test("returns an error when trying to create an Attribute Value Type EMailAddress without the right ending in the value for EMailAddress", function () {
+        const invalidEMailAdressCall = () => {
+            EMailAddress.from({
+                value: "Becker@gmx"
+            });
+        };
+        expect(invalidEMailAdressCall).toThrow(
+            new ParsingError(
+                "EMailAddress",
+                "value",
+                "Value does not match regular expression /^[a-z0-9!#$%&'*+/=?^_`{|}~-]+(?:[.][a-z0-9!#$%&'*+/=?^_`{|}~-]+){0,}@(?:[a-z0-9ÄäÖöÜüß](?:[a-zÄäÖöÜüß0-9-]{0,61}[a-zÄäÖöÜüß0-9]){0,1}[.])+[a-zÄäÖöÜüß0-9](?:[a-zÄäÖöÜüß0-9-]{0,61}[a-zÄäÖöÜüß0-9])$/i"
```

Comment:
Just a question out of interest: Why are the unicode terms, which originally appear in the regular expression (e.g. `\u00c4`), already replaced by the actual characters (e.g. `Ä`) in the tests for EMailAddress?

This seems not to be the case in the tests for the Website, where the unicode terms still occur in the regular expressions of the tests.

Reply:
In the RegExpConstructor you can insert a string than you have to use "" or a regexp there you use // and that's why in one case it translates the expression and in the other not, but you are right it should be traded in the same way and so I will change this.